felles/sikkerhet/src/main/java/no/nav/vedtak/sikkerhet/oidc/OidcTokenValidator.java

```diff
@@ -90,7 +91,10 @@ private OidcTokenValidatorResult validate(TokenString tokenHolder, int allowedCl
             if (error != null) {
                 return OidcTokenValidatorResult.invalid(error);
             }
-            return OidcTokenValidatorResult.valid(claims.getSubject(), claims.getExpirationTime().getValue());
```

Comment:
Du kan jo begynne forsiktig med å sjekke på TokenX - ikke sikker på om OpenAm/STS finner på å sette pid
Man kan pelle ut OpenIdProvider i  public OidcTokenValidator(OpenIDConfiguration config) og sette den for senere test

Reply:
Tenker du på at openam eller sts kan ha satt annen irrelevant/ukurant pid? Vi ser for så vidt i dev at tokenx nå inneholder en ukurant verdi i sub-feltet, så greit å være forsiktig.

Reply:
Skrivet om til å bare gjøre dette hvis tokenet er issued fra tokendings (altså tokens)